superset-frontend/src/components/WarningIconWithTooltip/index.tsx

```diff
@@ -39,7 +39,7 @@ function WarningIconWithTooltip({
       <Icons.AlertSolid
         iconColor={theme.colors.alert.base}
         iconSize={size}
-        css={{ marginRight: theme.gridUnit * 2 }}
+        css={{ marginRight: theme.gridUnit * 0.5 }}
```

Comment:
This change shouldn't be made in the component as it will affect many parts of the application. Could you make this adjustment in `TableSelector`?